UI/src/view/point_asset/trafficSignForm.js

```diff
@@ -43,6 +43,7 @@
       var additionalPanels = getProperties(asset.propertyData, "additional_panel");
       var checked = _.isEmpty(additionalPanels.values) ? '' : 'checked';
       var renderedPanels = checked ? renderAdditionalPanels(additionalPanels, collection) : '';
+      var roadSideInfo = asset.roadSideInfo ? "Kyllä" : "Ei";
```

Comment:
The variable name is not descriptive. Could be 'onOppositeSide' or something similar.

Reply:
trafficFlowContraryDirection

Reply:
oppositeSide or similar as Oskar suggested seems a better option